### PR TITLE
Mgmt docker and document

### PR DESCRIPTION
**Pull request by mlsmaycon**
- [x] use separated docker hub image repository
- [x] update documentation
- [x] update infrastructure_files
- [ ] Link docker hub documentation

### CODE REVIEW

File: management/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,3 @@
-FROM gcr.io/distroless/base:debug
+FROM gcr.io/distroless/base
 ENTRYPOINT [ "/go/bin/wiretrustee","management"]
```
Comment (andpar83) [inline management/Dockerfile:2]: I thought it would be a separate binary, no?

Comment (braginini) [reply management/Dockerfile:2]: Looks like it is @andpar83 
When I run `goreleaser --debug --snapshot --skip-publish --rm-dist` locally it produces a folder `dist/` with separate folders and each of them has a `wiretrustee` binary in it. 

├── goreleaserdocker332486772
├── goreleaserdocker417631529
├── goreleaserdocker460708675
├── goreleaserdocker464271775
├── goreleaserdocker819584114
├── wiretrustee_darwin_amd64
├── wiretrustee_linux_amd64
├── wiretrustee_linux_arm_6
├── wiretrustee_linux_arm64
**├── wiretrustee-mgmt_linux_amd64
├── wiretrustee-mgmt_linux_arm64**
└── wiretrustee_windows_amd64

When I run this:
`./dist/wiretrustee-mgmt_linux_amd64/wiretrustee --help` I get only management service-related output. 
Seems to be correct, right @mlsmaycon ?
```
Usage:
  wiretrustee [command]

Available Commands:
  help        Help about any command
  management  start Wiretrustee Management Server

Flags:
      --config string      Wiretrustee config file location to write new config to (default "/etc/wiretrustee/config.json")
  -h, --help               help for wiretrustee
      --log-level string    (default "info")

Use "wiretrustee [command] --help" for more information about a command.

```

Comment (mlsmaycon) [reply management/Dockerfile:2]: I am able to generate a different binary as well(e.g.  wiretrustee-mgmt ). Do you guys prefer that?

**Comment by mlsmaycon** [comment]
Docker hub doesn't allow tokens to update the description of a repo as described in: https://github.com/docker/hub-feedback/issues/1927 and https://github.com/docker/roadmap/issues/115

for now, I will keep the intended configuration here for future reference:

```shell 
      -
        name: Docker Hub Description for Management
        uses: peter-evans/dockerhub-description@v2
        with:
          username: ${{ secrets.DOCKER_USER }}
          password: ${{ secrets.DOCKER_TOKEN }}
          repository: wiretrustee/management
          readme-filepath: ./management/README.md
```

File: management/Dockerfile (hunk 1 above)
Comment (braginini) [reply management/Dockerfile:2]: @mlsmaycon yeah, I think this way is better

**Review by braginini — APPROVED** [review]

**Review by andpar83 — APPROVED** [review]

**Pull request merged**